tests/chocolatey-tests/choco-install.Tests.ps1

```diff
@@ -565,6 +565,10 @@ Describe "choco install" -Tag Chocolatey, InstallCommand {
             $XML.package.metadata.version | Should -Be "1.0.0"
         }
 
+        It "Outputs a warning message about side by side installs are deprecated" {

```

Comment:
Could we skip this if version is below 1.2.0-alpha? That will allow these tests to be bypassed with current stable builds of Chocolatey.